### DISCUSSION TITLE
Idées de projet

### DISCUSSION THREAD

**Discussion by rascar-capac** [rascar-capac/technocite-2021-lets-get-physical#1] [Ideas]
- jeu de foot/basket
  - il faut appliquer une force à une balle et viser des cibles
- angry birds 3D
  - il faut détruire des cibles protégées
  - avec un nombre de projectiles varié mais limité
  - on choisit la direction et l’intensité du tir
- tir à l’arc
  - on peut tirer des flèches dans une direction et avec une intensité
  - elles peuvent se planter dans une cible
- puzzle ball
  - il faut amener la boule à la sortie du puzzle
  - on peut déplacer et faire sauter la boule (en appliquant des forces)
  - des obstacles compliquent le parcours
- spider man se rend au travail
  - il faut se rendre à un autre endroit d’une map (ou tenir le plus longtemps possible)
  - les déplacements se font grâce à des toiles à coller sur des surfaces
  - on peut rétracter ses toiles et être attiré vers l’autre bout de la toile
  - on peut se détacher de ses toiles
- hit climb racing like
  - il faut arriver à la fin du circuit (en un temps imparti) en contrôlant le véhicule à travers un terrain accidenté
  - on peut accélérer et freiner
  - si la voiture se renverse, c’est perdu
- aspirateur/souffleur
  - tirer des flux d’air pour chasser des objets et les déplacer dans des endroits stratégiques, ou les aspirer et remplir un réservoir mais en éviter certains
- construire des ponts
  - on peut déplacer des pièces et les attacher entre elles pour former des structures plus ou moins solides sur lesquelles une boule doit passer
- découper des objets au sabrolaser <https://www.youtube.com/watch?v=BVCNDUcnE1o>
- jeu de bucheron <https://www.youtube.com/watch?v=ql4prUAasEg>
- soft bodies <https://www.youtube.com/watch?v=h_bSGoztB2g>
- cut the rope => exercice de chérif
- jeux classiques avec de la physique (tetris, space invaders, casse brique, etc)
- livreur de pizzas
  - déplacer un véhicule sans faire tomber des pizzas en équilibre placées dessus
- jenga
- surgeon simulator
- gang beasts

**Comment by Midonk** [comment]
# Ideas
- pilotage d'OVNI
- utilisation de portail (conservation des vitesses, ...)
- simulation balistiques (à la sniper elite avec des ralentis et tout :o )
- un universal sandbox
- un truc avec des explosion (c'est de la bonne physique ça)

# Notions physiques
- La gravité
- L'action, réaction
- la vitesse (linéaire ou orbitale)
- La lumière
- La température
- Les forces en tout genre...
- La poussée d'Archimède
- La propulsion
- L'électricité
- L'équilibre
- Transfert d'énergie
- L'électromagnétisme (les champs magnétiques et les aimants en somme)
- Le son et ondes vibratoires

# Références
[Physique: notions de base](https://cours-examens.org/images/An_2017_1/Etudes_superieures/Ondes/News/D1-2-Physique-Notions-base.pdf)

**Comment by niborobin19** [comment]
un flipper
